training/models.py

```diff
@@ -75,25 +75,27 @@ def forward(self, obs):
 
 
 class SafeLifePolicyNetwork(nn.Module):
-    def __init__(self, input_shape):
+    def __init__(self, input_shape, dense_depth=1):
```

Comment:
As per above, I think we should ax the depth code for this PR and keep it in a more experimental branch.